### PR TITLE
Provision a Sentry project automatically for newly deployed apps

### PR DESCRIPTION
**Pull request by jsierles**
### Change Summary

What and Why:

With this change, anew apps launched on Fly get a free Sentry project associated with them. This is the goal of a Fly.io partnership with Sentry to provide a free Sentry account to Fly.io customers for error tracking.

How:

This PR ensures the provisioning happens, and also sets the `SENTRY_DSN` so VMs will pick them up. 

### Documentation

- [ ] Fresh Produce
- [ ] In superfly/docs, or asked for help from docs team
- [ ] n/a

### CODE REVIEW

File: internal/command/deploy/deploy_first.go [hunk 1]
```diff
@@ -14,14 +18,46 @@ func (md *machineDeployment) provisionFirstDeploy(ctx context.Context, allocPubl
...
+
+	input := gql.SetSecretsInput{
+		AppId: md.app.ID,
+	}
+
+	fmt.Fprintf(md.io.Out, "Setting the following secrets on %s:\n", md.app.Name)
+
+	for key, value := range extension.Environment.(map[string]interface{}) {
+		input.Secrets = append(input.Secrets, gql.SecretInput{Key: key, Value: value.(string)})
+		fmt.Println(key)
```
Comment (dangra) [inline internal/command/deploy/deploy_first.go:53]: debug line?

File: internal/command/deploy/deploy_first.go [hunk 2]
```diff
@@ -14,14 +18,46 @@ func (md *machineDeployment) provisionFirstDeploy(ctx context.Context, allocPubl
...
+		return err
+	}
+
+	input := gql.SetSecretsInput{
+		AppId: md.app.ID,
+	}
+
+	fmt.Fprintf(md.io.Out, "Setting the following secrets on %s:\n", md.app.Name)
+
+	for key, value := range extension.Environment.(map[string]interface{}) {
```
Comment (dangra) [inline internal/command/deploy/deploy_first.go:51]: does casting as `map[stirng]string` not work?

**Review by dangra — COMMENTED** [review]
Looks good. It's a Draft PR, I guess you are still working on it.

File: internal/command/secrets/secrets.go [hunk 3]
```diff
@@ -78,8 +78,9 @@ func v1deploySecrets(ctx context.Context, app *api.AppCompact, release *api.Rele
 	return watch.Deployment(ctx, app.Name, release.EvaluationID)
 }
 
-func v2deploySecrets(ctx context.Context, app *api.AppCompact, release *api.Release, stage bool, detach bool) error {
+func DeploySecrets(ctx context.Context, app *api.AppCompact, stage bool, detach bool) error {
```
Comment (dangra) [inline internal/command/secrets/secrets.go:81]: seems not used, is this change needed?

Comment (jsierles) [reply internal/command/secrets/secrets.go:81]: Sorry, I didn't push recent changes, and now those changes have changed, so will re-request later!
